recognition/Keats-classifiers/assignment.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```

Comment:
Code header block should be added for all scripts/source files, author info, file description etc.